src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -119,6 +120,40 @@ def test_aks_create_and_update_with_managed_nat_gateway_outbound(self, resource_
                 'networkProfile.natGatewayProfile.managedOutboundIpProfile.count', 2),
         ])
 
+    @live_only()  # live only is required for test environment setup like `az login`
```

Comment:
It's ok to remove the `@live_only()` tag for `test_aks_create_and_update_with_managed_nat_gateway_outbound`, `test_aks_update_outbound_from_slb_to_natgateway`. I'll help generate the recording file for the latter case. For the former one, the recording file already exists.

Reply:
@MartinForReal

Reply:
Thanks!